### ISSUE TITLE
Cannot get dependencies in last 1.3.5 version

### ISSUE THREAD

**Issue by HeyShafty** [espresso3389/flutter_pdf_render#99]
Steps to reproduce :
- Create a new flutter project
- add dependency `pdf_render: ^1.3.5`
- Cannot `flutter pub get`

```text
Because every version of flutter_test from sdk depends on vector_math 2.1.2 and pdf_render >=1.3.5 depends on vector_math ^2.1.4, flutter_test from sdk is incompatible with pdf_render >=1.3.5.
So, because flutter_application_1 depends on both pdf_render ^1.3.5 and flutter_test from sdk, version solving failed.
pub get failed (1; So, because flutter_application_1 depends on both pdf_render ^1.3.5 and flutter_test from sdk, version solving failed.)
```
  
Sample `pubspec.yml`: 
```yml
name: flutter_application_1
description: A new Flutter project.

publish_to: 'none' # Remove this line if you wish to publish to pub.dev

version: 1.0.0+1

environment:
  sdk: '>=2.18.2 <3.0.0'

dependencies:
  flutter:
    sdk: flutter
  pdf_render: ^1.3.5

dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_lints: ^2.0.0

flutter:
  uses-material-design: true
```

**Comment by HeyShafty** [comment]
Related to https://github.com/espresso3389/flutter_pdf_render/commit/338d24f9f10954cd4da635a12ebeea6c8b993938#diff-8b7e9df87668ffa6a04b32e1769a33434999e54ae081c52e5d943c541d4c0d25R19 change

**Comment by espresso3389** [comment]
Fixed on ed6eb65ad940e4c42e991c35cb33742329fcdff2 ([1.3.6](https://pub.dev/packages/pdf_render/versions/1.3.6))
